internal/cli/debug_pprof.go

```diff
@@ -129,8 +129,10 @@ func (d *DebugPprofCommand) Run(args []string) int {
 
 	// Only take cpu and heap profiles by default
 	profiles := map[string]string{
-		"heap": "heap",
-		"cpu":  "cpu",
+		"heap":  "heap",
+		"cpu":   "cpu",
+		"trace": "trace",
```

Comment:
"trace" was removed from the default list of profiles because it takes too much space in the final output. That case is handled below at line 138.

Reply:
Will be taken care when merging the PR.

Reply:
Resolved ✅